certbot-nginx/certbot_nginx/obj.py

```diff
@@ -73,8 +88,10 @@ def fromstring(cls, str_addr):
                 ssl = True
             elif nextpart == 'default_server':
                 default = True
+            elif nextpart == "ipv6only=on":
```

Comment:
What do we want this variable to represent? "Was ipv6only=on explicitly set" is what it currently is. That's not the same as "is ipv6only on for this address", because the default varies by Nginx version.

Reply:
ipv6only=on can exist only once throughout Nginx configuration, having it explicitly set on two different server blocks would cause Nginx to error out.